ecosystem/typescript/sdk/examples/typescript/multisig_managed_coin.ts

```diff
@@ -0,0 +1,267 @@
+    let fungibleStore3 = await client.getAccountResource(getPrimaryStoreAddress(owner3, metadata), fungibleStore);
+    assert(parseInt((fungibleStore3?.data as any).balance) === 1_000);
+  }
+
+
+  // Step 5: Create another multisig transaction to burn fungible assets.
+  // ===========================================================================================
+  {
+    // :!:>section_6
+    const burnAccountsSerializer = new BCS.Serializer();
+    BCS.serializeVector([AccountAddress.fromHex(owner1.address()), AccountAddress.fromHex(owner2.address()), AccountAddress.fromHex(owner3.address())], burnAccountsSerializer);
+    const burnPayload = new MultiSigTransactionPayload(
+      EntryFunction.natural(
+        MODULE_ADDR + "::managed_fungible_asset",
+        "burn_from_primary_stores",
+        [],
+        [BCS.bcsToBytes(metadata), burnAccountsSerializer.getBytes(), BCS.serializeVectorWithFunc([1_000, 1_000, 1_000], "serializeU64")],
+      ),
+    );
+    const burnTx = await client.generateTransaction(owner2.address(), {
+      function: "0x1::multisig_account::create_transaction",
+      type_arguments: [],
+      arguments: [multisigAddress, BCS.bcsToBytes(burnPayload)],
+    });
+    await client.generateSignSubmitWaitForTransaction(owner2, burnTx.payload);
+    await approve(client, owner1, multisigAddress, 4);
+    await client.generateSignSubmitWaitForTransaction(
+      owner2,
+      new TransactionPayloadMultisig(new MultiSig(AccountAddress.fromHex(multisigAddress))),
+    );
+    // <:!:section_6
+    // Check the primary store balance of owner1, owner2 and owner3.
+    let fungibleStore1 = await client.getAccountResource(getPrimaryStoreAddress(owner1, metadata), fungibleStore);
+    assert(parseInt((fungibleStore1?.data as any).balance) === 0);
+    let fungibleStore2 = await client.getAccountResource(getPrimaryStoreAddress(owner2, metadata), fungibleStore);
+    assert(parseInt((fungibleStore2?.data as any).balance) === 0);
+    let fungibleStore3 = await client.getAccountResource(getPrimaryStoreAddress(owner3, metadata), fungibleStore);
+    assert(parseInt((fungibleStore3?.data as any).balance) === 0);
+  }
+})();
+
+const approve = async (
+  client: AptosClient,
+  owner: AptosAccount,
+  multisigAddress: string,
+  transactionId: number,
+) => {
+  let approveTx = await client.generateTransaction(owner.address(), {
+    function: "0x1::multisig_account::approve_transaction",
+    type_arguments: [],
+    arguments: [multisigAddress, transactionId],
+  });
+  await client.generateSignSubmitWaitForTransaction(owner, approveTx.payload);
+};
+
+const getNamedObjectAddress = (owner: AptosAccount, seed: string): TxnBuilderTypes.AccountAddress => {
+  const hash = sha3Hash.create();
+  hash.update(BCS.bcsToBytes(AccountAddress.fromHex(owner.address())));
+  hash.update(seed);
+  hash.update(new Uint8Array([0xFE]));
+  return AccountAddress.fromHex(Buffer.from(hash.digest()).toString("hex"));
```

Comment:
why not simply return `HexString`?
```suggestion
  return HexString.fromBuffer(Buffer.from(hash.digest()));
```